test/core/formatError.test.js

```diff
@@ -127,12 +137,65 @@ suite("formatError", function() {
       "  1001 | \n" +
       "  1002 | \n" +
       "\n" +
-      "Expected one of the following: \n\n" +
+      "Expected one of the following: \n" +
+      "\n" +
       "'\n', 'c'\n";
 
     var input = new Array(999).join("\n") + "\nd" + new Array(10).join("\n");
     var answer = Parsimmon.formatError(input, parser.parse(input));
-    // console.log(answer);
+
+    assert.deepEqual(answer, expectation);
+  });
+
+  test("byte buffer error with multi-line input", function() {
+    var parser = Parsimmon.seq(
+      Parsimmon.Binary.byte(0x00).many(),
+      Parsimmon.Binary.byte(0x01),
+      Parsimmon.Binary.byte(0x00).many()
+    );
+
+    var expectation =
+      "\n" +
+      "-- PARSING FAILED --------------------------------------------------\n" +
+      "\n" +
+      "   2 | 0 0  0 0 0 0 0 0 0 0\n" +
+      "   3 | 0 0  0 0 0 0 0 0 0 0\n" +
+      "   4 | 0 0  0 0 0 0 0 0 0 0\n" +
+      "   5 | 0 0  0 0 0 0 0 0 0 0\n" +
+      "   6 | 0 0  0 0 0 0 0 0 0 0\n" +
+      ">  7 | 0 0 ff 0 0 0 0 0 0 0\n" +
+      "     |     ^^\n" +
+      "   8 | 0 0  0 0 0 0 0 0 0 0\n" +
+      "   9 | 0 0  0 0 0 0 0 0 0 0\n" +
+      "  10 | 0 0  0 0 0 0 0 0 0 0\n" +
+      "  11 | 0 0  0 0 0 0 0 0 0 0\n" +
```

Comment:
i think each hex byte should be written as two digits, like `00` instead of `0` so that it is easier to read

Reply:
Good point, I will address this asap! 👍